admin_guide/scheduler.adoc

```diff
@@ -668,3 +668,163 @@ ifdef::openshift-enterprise[]
 # systemctl restart atomic-openshift-master
 ----
 endif::[]
+
+[[controlling-pod-placement-projects]]
+=== Control Pod Placement to Projects
```

Comment:
Dunno why, but this isn't rendering right in the Github view thing:

https://github.com/mburke5678/openshift-docs/blob/a23b6c4f3eef11467533501a601a35089d58deb1/admin_guide/scheduler.adoc

(Scroll down the bottom)

Might be because the ifdefs above lead onto a + that would render well enough.

Reply:
@bfallonf Odd. Looked fine in the asciibinder build preview.